### PR TITLE
Scala breakout

### PR DESCRIPTION
**Pull request by marconilanna**
@matthandlersux adds a note about `scala.collection.breakOut` and links to the "broken windows" interview (recommended reading).

something must have tripped the diff off, i just moved the enum item to a different section. feel free to ignore the two big blocks, nothing changed there.

### CODE REVIEW

File: scala/README.md [hunk 1]
```diff
@@ -868,7 +880,9 @@ Additional Remarks
 
     [Martin Fowler](http://www.martinfowler.com/bliki/AnemicDomainModel.html)
 
-0. Always remember Tip 4: "Do not Live with Broken Windows: Fix each as soon as it is discovered." - The Pragmatic Programmer, Andrew Hunt and David Thomas.
+0. Always remember Tip 4: "[Do not Live with Broken Windows](http://www.artima.com/intv/fixit.html): Fix each as soon as it is discovered." - The Pragmatic Programmer, Andrew Hunt and David Thomas.
+
+Part II, [Orthogonality and the DRY Principle](http://www.artima.com/intv/dry.html), is also an interesting reading. [Full interview](http://www.artima.com/intv/plain.html).
```
Comment (matthandlersux) [inline scala/README.md]: "is also an interesting read*" or "is also interesting reading" (the former is more common)

**Comment by matthandlersux** [comment]
LGTM

**Comment by marconilanna** [comment]
@matthandlersux comments addressed

**Comment by matthandlersux** [comment]
:ship:

**Pull request merged**